src/sentry/sentry_metrics/querying/api.py

```diff
@@ -29,6 +29,17 @@
 
 # Snuba can return at most 10.000 rows.
 SNUBA_QUERY_LIMIT = 10000
+# Intervals in seconds which are used by the product to query data.
+DEFAULT_QUERY_INTERVALS = [
+    60 * 60 * 24,  # 1 day
+    60 * 60 * 12,  # 12 hours
+    60 * 60 * 4,  # 4 hours
+    60 * 60 * 2,  # 2 hours
+    60 * 60,  # 1 hour
+    60 * 30,  # 30 min
+    60 * 5,  # 5 min
+    60,  # 1 min
```

Comment:
There's also a special one - 10s. It's defined outside of the list that I sent you.

https://github.com/getsentry/sentry/blob/40d181ca536587d6c649ff964123a556dc79e88c/static/app/utils/metrics/index.tsx#L237-L239

Reply:
I can add it in a follow up PR, so that we see how it performs in prod with these settings.

Reply:
It's also dependent on the dataset, since for generic metrics we have at least 10s of granularity but for others session metrics not.